components/main_menu/index.tsx

```diff
@@ -78,6 +89,9 @@ function mapStateToProps(state: GlobalState) {
         guestAccessEnabled: config.EnableGuestAccounts === 'true',
         canInviteTeamMember,
         isFirstAdmin: isFirstAdmin(state),
+        isCloudFreeEnabled,
```

Comment:
This is where we may change this to just `isCloud` and check at this level the cloud license and the feature flag